packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -291,7 +291,8 @@ export function DAppsExplorerScreen() {
 }
 
 function parseResultIntoSections(categoriesWithDapps: CategoryWithDapps[]): SectionData[] {
-  return categoriesWithDapps.map((category) => ({
+  const filtered = categoriesWithDapps.filter((category) => category.dapps.length)
```

Comment:
super nit: the first input of this `parseResultIntoSections` function is called `categoriesWithDapps` which made me think it was already filtering the empty categories, could we rename this to just `categories` or something? that frees up the namespace to call this variable `categoriesWithDapps` :D